shared/components/Animation/Reveal.jsx

```diff
@@ -13,9 +13,14 @@ const transitionStyles = height => {
     height: `${height}px`,
   }
 
+  const hiddenContent = {
+    visibility: 'hidden',
+  }
+
```

Comment:
Let's hoist `hiddenContent` out of the `transitionStyles` function since it never changes